### ISSUE TITLE
post process

### ISSUE THREAD

**Issue by yair-6over6** [sterlingrpi/ssd_tflite_tpu#2]
Hi, thank you for this repo,  very useful!
I have a question / request:

I can see that the tflite convert here is without the post process logic.
Is there a simple way pack it into the converted tflite file as extra layer? 
(So I won't have to write it on my app java code)
Thanks!

**Comment by FSet89** [comment]
Hi, did you find a solution?

**Comment by wakasturner** [comment]
Hey thanks for a great repo👍
I am also trying to pack post processing when converting to tflite, any pointers would be really appreciated.

@FSet89 @yair-6over6 
If any of you guys had found the solution, appreciate if you shared your solution🙌

**Comment by sterlingrpi** [comment]
I don't know if post processing steps, like anchor boxes, would be supported by tflite and the tpu converter.

**Comment by wakasturner** [comment]
@sterlingrpi 
Thanks for the reply, 
the [official](https://github.com/tensorflow/models/) TensorFlow repos ssd conversion do support packing of post processing functions in tflite.
I have had trained a model using your repo and was able to convert it to tflite but am stuck on how to add post processing.

**Comment by sterlingrpi** [comment]
Wow, interesting. And the official Tensorflow repo doesn't support training? I'll try to take a lot at what they did. Maybe we can swap the model somewhere before the conversation.

The problem I was having was finding an SSD repo that supported training on a custom dataset and converting to tflite and tpu. It was a couple years ago. Maybe someone else came up with a solution or the tflite and tpu converter improved. Haven't tried it since 2020 when I was trying to keep the squirrel out of my bird feeder.

Thank you for using my repo. I hope it was helpful. Off topic question. What did you use to label your dataset?

**Comment by wakasturner** [comment]
Yeah, you are right, i just didn't wanted to retrain the whole model. lol
As an alternative I am thinking of using the already trained model (using your repo) and using it as a finetuning checkpoint in the official repo trainer.  
I used Microsoft's [vott](https://github.com/microsoft/VoTT) to label and export the dataset to Pascal VOC.
Your repo helped me a lot to get jump started on my project. Thanks again. 👍
